core/gpslocation.cpp

```diff
@@ -36,6 +37,9 @@ GpsLocation::GpsLocation(void (*showMsgCB)(const char *), QObject *parent) :
 	userAgent = getUserAgent();
 	(void)getGpsSource();
 	loadFromStorage();
+
+	// register changes in time threshold
+	connect(qPrefLocationService::instance(), SIGNAL(qPrefLocationService::time_thresholdChanged()), this, SLOT(setGpsTimeThreshold(int seconds)));
```

Comment:
the code doesn't seem to match the commit message. Yes, you now listen to a signal, but where is the "do not call" part?
Or am I misunderstanding the commit message?